.circleci/config.yml

```diff
@@ -174,6 +174,23 @@ jobs:
           name: Publish to npm
           command: yarn release:next --yes
 
+  publish-docs:
+    executor: env-node10
+    steps:
+      - checkout
+      - run:
+          name: install aws cli
```

Comment:
Optional: we can be cool and try https://circleci.com/orbs/registry/orb/circleci/aws-s3.